docs/Bootloader_Entry.md

```diff
@@ -0,0 +1,119 @@
+# Bootloader Entry
+
+Klipper can be instructed to reboot into a [Bootloader](Bootloaders.md) in one
+of the following ways:
+
+## Requesting the bootloader
+
+### Virtual Serial
+
+If a virtual (USB-ACM) serial port is in use, pulsing DTR while at 1200 baud
+will request the bootloader.
+
+#### Python (with `flash_usb`)
+
+To enter the bootloader using python (using `flash_usb`):
+
+```shell
+> cd klipper/scripts
+> python3 -c 'import flash_usb as u; u.enter_bootloader("<DEVICE>")'
+Entering bootloader on <DEVICE>
+```
+
+Where `<DEVICE>` is your serial device, such as
+`/dev/serial.by-id/usb-Klipper[...]` or `/dev/ttyACM0`
+
+Note that if this fails, no output will be printed, success is indicated by
+printing `Entering bootloader on <DEVICE>`.
+
+#### Picocom
+
+```shell
+picocom -b 1200 <DEVICE>
+<Ctrl-A><Ctrl-P>
+```
+
+Where `<DEVICE>` is your serial device, such as
+`/dev/serial.by-id/usb-Klipper[...]` or `/dev/ttyACM0`\
+`<Ctrl-A><Ctrl-P>` means
+holding `Ctrl`, pressing and releasing `a`, pressing and releasing `p`, then
+releasing `Ctrl`
+
+### Physical serial
+
+If a physical serial port is being used on the MCU (even if a USB serial adapter
+is being used to connect to it), sending the string
+`<SPACE><FS><SPACE>Request Serial Bootloader!!<SPACE>~`.
+
+`<SPACE>` is an ASCII literal space, 0x20.\
+`<FS>` is the ASCII File Separator,
+0x1c.
+
+Note that this is not a valid message as per the
+[MCU Protocol](Protocol.md#micro-controller-interface), but sync characters(`~`)
+are still respected.\
```

Comment:
I noticed several instances in this file where the line ends in a backslash (`\`).  Why is this?

Reply:
This causes a line break to be inserted when rendering as HTML, otherwise it takes a blank line, or two spaces at the end of a line.

When I wrote the file, I had used double spaces, but running it through `mdformat` swapped them for backslashes.

Reply:
Okay, thanks.  I'd recommend inserting blank lines instead.  Ideally the documentation will be generally readable in text, on github, and on the klipper3d.org site - using a backslash seems subtle and I fear it may not render well on all three locations.

-Kevin

Reply:
Updated